FluentValidation.AutoValidation.Shared/src/Extensions/TypeExtensions.cs

```diff
@@ -7,18 +8,11 @@ public static class TypeExtensions
     {
         public static bool IsCustomType(this Type? type)
         {
-            var builtInTypes = new[]
-            {
-                typeof(string),
-                typeof(decimal),
-                typeof(DateTime),
-                typeof(DateTimeOffset),
-                typeof(TimeSpan),
-                typeof(Guid),
-                typeof(Enum)
-            };
-
-            return type != null && type.IsClass && !type.IsEnum && !type.IsValueType && !type.IsPrimitive && !builtInTypes.Contains(type);
+            return type != null && type.IsClass 
+                && !type.IsEnum && !type.IsValueType 
+                && !type.IsPrimitive
+                && type != typeof(string)
+                && !type.GetInterfaces().Contains(typeof(IEnumerable));
```

Comment:
I guess that should be considered as breaking change. If someone is right now using e.g. List<T> as Body parameter, after this change validation will stop working for this parameter.

Reply:
Yes you're right. You can use any collection props for custom types and add FluentValidation validators for them, thank you for mention it in your issue
I relaxed checking by deleting the IEnumerable check